src/Environment.hpp

```diff
@@ -1,9 +1,8 @@
 #ifndef ENVIRONMENT_HPP
```

Comment:
Is Environment the best possible name for this class? Of which kind of environment are we talking here?

Reply:
renamed metaVariables (file, methods, everything)
The "environment variables" are called "meta-variables" in the CGI